frontend/src/pages/connectionTypes/ConnectionTypesTable.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { FilterDataType, initialFilterData } from '~/pages/connectionTypes/const';
+import { connectionTypeColumns } from '~/pages/connectionTypes/columns';
+import DashboardEmptyTableView from '~/concepts/dashboard/DashboardEmptyTableView';
+import ConnectionTypesTableRow from '~/pages/connectionTypes/ConnectionTypesTableRow';
+import ConnectionTypesTableToolbar from '~/pages/connectionTypes/ConnectionTypesTableToolbar';
+import { ConnectionTypeConfigMapObj } from '~/concepts/connectionTypes/types';
+import { Table } from '~/components/table';
+
+interface ConnectionTypesTableProps {
+  connectionTypes: ConnectionTypeConfigMapObj[];
+  onUpdate: () => void;
+}
+
+const ConnectionTypesTable: React.FC<ConnectionTypesTableProps> = ({
+  connectionTypes,
+  onUpdate,
+}) => {
+  const [filterData, setFilterData] = React.useState<FilterDataType>(initialFilterData);
+  const onClearFilters = React.useCallback(() => setFilterData(initialFilterData), [setFilterData]);
+
+  const filteredConnectionTypes = connectionTypes.filter((connectionType) => {
+    const keywordFilter = filterData.Keyword?.toLowerCase();
+    const createFilter = filterData['Created by']?.toLowerCase();
+
+    if (
+      keywordFilter &&
+      !(
+        connectionType.metadata.annotations?.['openshift.io/display-name'] ||
+        connectionType.metadata.name
+      )
+        .toLowerCase()
+        .includes(keywordFilter) &&
+      !connectionType.metadata.annotations?.['openshift.io/description']
+        ?.toLowerCase()
+        .includes(keywordFilter)
+    ) {
+      return false;
+    }
+
+    return (
+      !createFilter ||
+      (connectionType.metadata.annotations?.['opendatahub.io/username'] || 'unknown')
+        .toLowerCase()
+        .includes(createFilter)
+    );
+  });
+
+  const resetFilters = () => {
+    setFilterData(initialFilterData);
+  };
+
+  return (
+    <>
```

Comment:
Kinda a nit and I don't know if it matters, but you can remove the fragment `<>` and just return the `<Table>` directly.

Everything else looks good to me